packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,138 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: c.$grid * 6;
+$w: c.$grid * 15;
+$labelMargin: c.$grid * 4;
+$trackBr: 22px;
+$trackBgOn: #1dbc22;
+$trackBgOff: c.$colorNeutralLight;
+$knobRight: $w * 0.6;
+$knobBgOn: c.$colorNeutralWhite;
+$knobBgOff: c.$colorNeutralLighter;
+$knobR: $h * 0.9;
```

Comment:
In figma, the knob radius is `20px`. It is `5 * $grid`. `$h * 0.9` is `5.4 * $grid`. Should we use `5 * $grid`?

Reply:
yup cleaner. implemented.